Cargo.toml

```diff
@@ -1,6 +1,9 @@
 [package]
 name = "rpch-crypto"
+description = "This crate implements the RPCh Crypto protocol as defined by the specs in the parent repository. The implementation is WASM compatible and also exposes a TypeScript API via wasm-bindgen."
 version = "0.3.3"
+license = "LGPL-3.0"
```

Comment:
Shouldn't it be GPL?

Reply:
No, the licence we use is LGPL across RPCh and HOPR

Reply:
In HOPR we're using GPL JFYI.